### PR TITLE
duckstation: unstable-2023-04-14 -> unstable-2023-09-30

### PR DESCRIPTION
**Pull request by AndersonTorres**
## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by qubitnano** [comment]
Result of `nixpkgs-review pr 258661` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>duckstation</li>
  </ul>
</details>

**Comment by qubitnano** [comment]
Thank you this works great

duckstation can use libbacktrace now if that's useful

File: pkgs/applications/emulators/duckstation/default.nix [hunk 1]
```diff
@@ -61,9 +78,9 @@ stdenv.mkDerivation {
   ++ cubeb.passthru.backendLibs;
 
   cmakeFlags = [
-    "-DUSE_DRMKMS=ON"
```
Comment (SuperSamus) [inline pkgs/applications/emulators/duckstation/default.nix:64]: If I'm not mistaken, with that removed, `mesa` can be removed from the dependencies.

Comment (AndersonTorres) [reply pkgs/applications/emulators/duckstation/default.nix:64]: duckstation searches for EGL.

Comment (SuperSamus) [reply pkgs/applications/emulators/duckstation/default.nix:64]: I tried removing `mesa`, EGL is found anyway:
```
-- Found EGL: /nix/store/8gp0sr5qn61f17z8vza7nm32kshbj679-libGL-1.6.0-dev/include (found version "1.5") 
```

Comment (AndersonTorres) [reply pkgs/applications/emulators/duckstation/default.nix:64]: How precisely? libGL is not an explicit input.

Comment (SuperSamus) [reply pkgs/applications/emulators/duckstation/default.nix:64]: It's from https://github.com/NixOS/nixpkgs/blob/c0d8c84f4024a3b2b6a7b648a3b692532bd84a2c/pkgs/development/libraries/qt-6/modules/qtbase.nix#L173

Comment (AndersonTorres) [reply pkgs/applications/emulators/duckstation/default.nix:64]: Nice.

**Pull request merged**